src/app/public/modules/action-button/action-button-container.component.ts

```diff
@@ -52,6 +52,9 @@ import {
   selector: 'sky-action-button-container',
   styleUrls: ['./action-button-container.component.scss'],
   templateUrl: './action-button-container.component.html',
+  providers: [
+    SkyActionButtonAdapterService
```

Comment:
Now that the adapter is provided by the component, we no longer need to generate a renderer using the `Renderer2Factory`.
Instead of this:
```
private renderer: Renderer2;

constructor(
  private rendererFactory: RendererFactory2
) {
  this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
}
```

We can now do this:
```
constructor(
  private renderer: Renderer2
) {}
```